stl/inc/atomic

```diff
@@ -1993,6 +2309,40 @@ struct atomic_flag { // flag with test-and-set semantics
 
     constexpr atomic_flag() noexcept = default;
 
+#if _HAS_CXX20
+    bool test(const memory_order _Order = memory_order::seq_cst) const noexcept {

```

Comment:
Unlike `test_and_set`, `test` is a pure observer, so it should be `_NODISCARD`, right? Ditto below, including non-member functions.